### DISCUSSION TITLE
Prevent other projects showing in the current project's source control window

### DISCUSSION THREAD

**Discussion by babakfp** [gitkraken/vscode-gitlens#1822] [Q&A] [answered]
Hi
![Screenshot 2022-02-12 165252](https://user-images.githubusercontent.com/44144724/153733714-a1f51e3c-8965-4b66-a73a-eaad432de395.png)
Let's say I open a few files from different projects that have uncommitted changes. What happens is that they show up in my current project's source control window:
![Screenshot 2022-02-12 165252](https://user-images.githubusercontent.com/44144724/153733750-4cb390f0-264d-43a5-9167-1680fcd1f0ba.png)
If it's possible I want to disable this behavior. It's driving me crazy😁
Thanks

**Comment by eamodio** [comment, marked as answer]
That is a feature of VS Code itself. You can [open an issue](https://github.com/microsoft/vscode/issues/new?assignees=&labels=&template=bug_report.md) on the VS Code repository.
